src/popup/components/screenshots.tsx

```diff
@@ -3,29 +3,69 @@ import * as React from 'react'
 type ScreenshotsProps = {
   feedback: FeedbackState
   startEditingScreenshot(payload: { screenshotIndex: number }): void
+  clickDeleteScreenshot(payload: { screenshotIndex: number }): void
+  deleteScreenshotDisabled: boolean
 }
 
 type ScreenshotThumbnailProps = {
   screenshot: Screenshot
   startEditingScreenshot(): void
+  clickDeleteScreenshot(): void
+  deleteScreenshotDisabled: boolean
 }
 
-export function ScreenshotThumbnail({ screenshot, startEditingScreenshot }: ScreenshotThumbnailProps) {
+function CloseButton({ onClick }: { onClick: () => void }): JSX.Element {
   return (
-    <>
-      <img className="screenshot-thumbnail" src={screenshot.uri} />
-      <button onClick={startEditingScreenshot}>Edit</button>
-    </>
+    <button className="close-button" onClick={onClick}>
```

Comment:
Can we make this into an `svg-btn` somehow?

Reply:
I can export the SVG from Figma but why?